extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -684,6 +705,26 @@ export class AmpConsent extends AMP.BaseElement {
   getIsPromptUiOnForTesting() {
     return this.isPromptUIOn_;
   }
+
+  /**
+   * If consentString is undefined or invalid, don't
+   * include any metadata in update.
+   * @param {*} metadata
+   * @param {string=} consentString
+   * @return {Object=}
+   */
+  configureMetadataByConsentString(metadata, consentString) {
```

Comment:
nit: This can be a private method

Reply:
Is there a unit test to cover the check? I couldn't find it.

Reply:
oops I forgot to do that :(